sanitizer/_size.js

```diff
@@ -1,36 +1,54 @@
 const _ = require('lodash');
-const MIN_SIZE = 1;
-const MAX_SIZE = 40;
-const DEFAULT_SIZE = 10;
+const peliasConfig = require('pelias-config').generate().api;
+//Allow custom min, max and default sizes
+const MIN_SIZE = peliasConfig.minSize || 1;
+const MAX_SIZE = peliasConfig.maxSize || 40;
+let default_size_init;
```

Comment:
maybe assign this to 10 to start so the Pelias defaults are more clearly listed in one place?